### PR TITLE
README: add suggestion to contribute with requested pages

### PR DESCRIPTION
**Pull request by CleanMachine1**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Review by EmilySeville7cfg — APPROVED** [review]
LGTM 👍🏻

**Review by mfrw — APPROVED** [review]
LGTM :)
Thanks @CleanMachine1

File: README.md [hunk 1]
```diff
@@ -81,6 +81,7 @@ Some ways to contribute include:
 
 - Adding your favorite command which isn't covered.
 - Adding examples or improving the content of an existing page.
+- Searching our [help wanted](https://github.com/tldr-pages/tldr/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) label for pages to make and contribute.
```
[the remark this answers is not in the document]
Comment (CleanMachine1) [reply README.md:84]:
```suggestion
- Contributing requested pages from our issues with the [help wanted](https://github.com/tldr-pages/tldr/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) label.
```

[the remark this answers is not in the document]
Comment (sbrl) [reply README.md]: All of these work I think.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**